website/docs/reference/appsmith-framework/widget-actions/intervals-time-events.md

```diff
@@ -2,35 +2,82 @@
 sidebar_position: 9
 ---
 
-# Intervals (Time events)
-
-Interval events (Timing functions in JavaScript) allow the user to periodically run APIs and DB queries. You can configure these by using the setInterval and clearInterval functions.
+# Intervals (Time Events)
 
+Interval events (timing functions in JavaScript) allow the user to periodically run API and DB queries. You can configure these by using the `setInterval` and `clearInterval` functions.
 
 <VideoEmbed host="youtube" videoId="ByE3aqlQ1pE" title="How To Use The SetInterval & ClearInterval Functions To Auto-Update Widgets" caption="How To Use The SetInterval & ClearInterval Functions To Auto-Update Widgets"/>
 
-
-
 ## setInterval()
 
-The setInterval() method executes a code snippet with a fixed time interval between the calls, i.e, it executes a trigger callback at a given interval. Here’s how you can use it while dynamically binding queries onto widgets:
+`setInterval()` executes a callback function with a fixed time interval between the calls. This function is analogous to the [JavaScript `setInterval` function](https://developer.mozilla.org/en-US/docs/Web/API/setInterval).
```

Comment:
do we have to mention that it is the same as the browser's function and link to the docs there?  Even if we are, let's not say JavaScript function, instead 'Window setInterval() method' 

Same for the other two functions in this page

Reply:
rm'd the bits about the JS functions since a couple of them do have some differences to how they work -- otherwise should be good to go :)